packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -668,6 +668,19 @@ describe('RPC Server Acceptance Tests', function () {
         });
     });
 
+    describe('eth_feeHistory', () => {
```

Comment:
Nit: Lets rename this to `Gas Price related RPC endpoints` and include the acceptance tests for `eth_gasPrice` as-well :)